connectivity/connectivity-demos-test/src_test/rest/bpm/TestAsyncInvocation.java

```diff
@@ -3,14 +3,22 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
 
+import org.awaitility.Awaitility;
```

Comment:
number one java library to do assertions with a timeout ... without having to implement retry and timeout checking on your own. :nerd_face: